examples/medplum-demo-bots/esbuild-script.mjs

```diff
@@ -22,7 +24,7 @@ const esbuildOptions = {
     '.ts': 'ts', // Load TypeScript files
   },
   resolveExtensions: ['.ts', '.js'],
-  external: botLayerDeps,
+  external: [...botLayerDeps, ...additionalExternals],
```

Comment:
Do you recall why this was necessary?  Isn't `twilio` now in `botLayer.dependencies` ?

Does this only apply to older versions before `twilio` was added?

Reply:
Actually, you are right, this was based on my setup when I was testing the bundle before we added twilio. Upgrading to the latest Medplum version shouldn't require this. Should I remove these changes or add some kind o warning about older versions?